test/scripts/traffic-flow-tests.sh

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Set default values
+export KUBECONFIG="${KUBECONFIG:-${HOME}/ovn.conf}"
+export OCI_BIN="${KIND_EXPERIMENTAL_PROVIDER:-docker}"
+export TFT_TEST_IMAGE="ghcr.io/wizhaoredhat/ocp-traffic-flow-tests:latest"
+export TRAFFIC_FLOW_TESTS_DIRNAME="ocp-traffic-flow-tests"
+export TRAFFIC_FLOW_TESTS_REPO="https://github.com/wizhaoredhat/ocp-traffic-flow-tests.git"
+export TRAFFIC_FLOW_TESTS_COMMIT="6a3a529012a514887a7032d99aed72f1a782b0dc"
+export TRAFFIC_FLOW_TESTS="${TRAFFIC_FLOW_TESTS:-'1-3'}"
+
+export SCRIPT_DIR="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)"
+cd "$(git rev-parse --show-toplevel)/.."
+export TOP_DIR="$(pwd)"
+export TRAFFIC_FLOW_TESTS_FULL_PATH="${TOP_DIR}/${TRAFFIC_FLOW_TESTS_DIRNAME}"
+
+log() {
+    local level="$1"
+    shift
+    echo "[$(date '+%Y-%m-%d %H:%M:%S')] [$level] $*"
+}
+
+error_exit() {
+    log "ERROR" "$*"
+    exit 1
+}
+
+check_dependencies() {
+    local dependencies=(sed pip git kubectl kind "${OCI_BIN}")
+    for cmd in "${dependencies[@]}"; do
+        if ! command -v "${cmd}" &> /dev/null; then
+            error_exit "Dependency not met: ${cmd}"
+        fi
+    done
+}
+
+usage() {
+    echo "Usage: $(basename "$0") {setup|run}"
+}
+
+create_config_file_defaults() {
+    local file_name="${1:-config.yaml}"
+
+    cat <<EOT > "$file_name"
+tft:
+  - name: "Github Workflow Test"
+    namespace: "default"
+    test_cases: "TBD"
+    duration: "10"
+    connections:
+      - name: "Connection_1"
+        type: "iperf-tcp"
+        instances: 1
+        server:
+          - name: "ovn-worker"
+            persistent: "false"
+            sriov: "false"
+        client:
+          - name: "ovn-worker2"
+            sriov: "false"
+        plugins: []
+kubeconfig: "TBD"
+kubeconfig_infra:
+EOT
+}
+
+process_test_results() {
+    local result_file="$1"
+
+    log "INFO" "Processing test results from: ${result_file}"
+    python evaluator.py eval-config.yaml ${result_file} ${result_file}.evaluated || error_exit "Evaluation failed."
+    ./print_results.py ${result_file}.evaluated || error_exit "Results evaluation failed."
```

Comment:
if you use latest version of ocp-tft, you no longer need to call evaluator.

For one, you can already do the evaluation when running the test by passing the second command line argument `eval-config.yaml` to `tft.py`. However, that config now can also be empty (or omitted).

As there is nothing to evaluate, you only need to run `./tft.py config.yaml -o "$OUTPUT_BASE"` and then `./print_results "${OUTOUT_BASE}000.json"`

Note that previously, `print_results.py` only could have operated on the output file from `evaluator.py`. That is no longer the case, there is now only one file format. The format that `tft.py` writes is the same that `evaluator.py`/`print_results.py` take as input (and the same that `evaluator.py` also outputs)). Hence, you don't need to call `evaluator.py`, especially since you don't have any eval-config.py that you'd want to use.